src/networking.c

```diff
@@ -3855,7 +3861,7 @@ int handleClientsWithPendingReadsUsingThreads(void) {
             continue;
         }
 
-        processInputBuffer(c);
+        serverAssert(processInputBuffer(c) == C_OK);
```

Comment:
Let's add a comment stating why we can assert that here.

Reply:
Good that you wrote that. Going over the logic again I think it might actually return a `C_ERR`. I'll fix the code.